pkg/graph/schema/types/plan_data_exchange_approach.graphql

```diff
@@ -0,0 +1,89 @@
+enum DataToCollectFromParticipants {
+  BANKING_INFORMATION_TO_MAKE_NON_CLAIMS_BASED_PAYMENTS
+  CLINICAL_DATA
+  COLLECT_BIDS_AND_PLAN_INFORMATION
+  COOPERATIVE_AGREEMENT_APPLICATION
+  DECARBONIZATION_DATA
+  EXPANDED_DEMOGRAPHICS_DATA
+  FEE_FOR_SERVICE_CLAIMS_AND_APPLY_MODEL_RULES
+  LEARNING_SYSTEM_METRICS
+  PARTICIPANT_AGREEMENT
+  PARTICIPANT_AGREEMENT_LETTER_OF_INTENT
+  PARTICIPANT_AGREEMENT_REQUEST_FOR_APPLICATION
+  PARTICIPANT_REPORTED_DATA
+  PARTICIPANT_REPORTED_QUALITY_MEASURES
+  PROVIDER_PARTICIPANT_ROSTER
+  REPORTS_FROM_PARTICIPANTS
+  SOCIAL_DETERMINANTS_OF_HEALTH
+  SURVEY
+  OTHER
+}
+
+enum DataToSendToParticipants {
+  DATA_FEEDBACK_DASHBOARD
+  NON_CLAIMS_BASED_PAYMENTS
+  OPERATIONS_DATA
+  PARTIALLY_ADJUSTED_CLAIMS_DATA
+  RAW_CLAIMS_DATA
+  DATA_WILL_NOT_BE_SENT_TO_PARTICIPANTS
+}
+
+enum AnticipatedMultiPayerDataAvailabilityUseCase {
+  MORE_COMPETENT_ALERT_DISCHARGE_TRANSFER_NOTIFICATION
+  SUPPLY_MULTI_PAYER_CLAIMS_COST_UTIL_AND_QUALITY_REPORTING
+  FILL_GAPS_IN_CARE_ALERTING_AND_REPORTS
+}
+
+enum MultiSourceDataToCollect {
+  COMMERCIAL_CLAIMS
+  LAB_DATA
+  MANUFACTURER
+  MEDICAID_CLAIMS
+  MEDICARE_CLAIMS
+  PATIENT_REGISTRY
+  OTHER
+}
+
+type PlanDataExchangeApproach {
+  id: UUID!
+  modelPlanID: UUID!
+  modelPlan: ModelPlan!
+
+  dataToCollectFromParticipants: [DataToCollectFromParticipants!]!
+  dataToCollectFromParticipantsReportsDetails: String
+  dataToCollectFromParticipantsOther: String
+  dataWillNotBeCollectedFromParticipants: Boolean!
+  dataToCollectFromParticipantsNote: String
+
+  dataToSendToParticipants: DataToSendToParticipants
+  dataToSendToParticipantsNote: String
+
+  doesNeedToMakeMultiPayerDataAvailable: YesNoType
+  anticipatedMultiPayerDataAvailabilityUseCase: AnticipatedMultiPayerDataAvailabilityUseCase
+  doesNeedToMakeMultiPayerDataAvailableOther: String
+  doesNeedToMakeMultiPayerDataAvailableNote: String
+
+  doesNeedToCollectAndAggregateMultiSourceData: YesNoType
+  multiSourceDataToCollect: MultiSourceDataToCollect
+  multiSourceDataToCollectOther: String
+  doesNeedToCollectAndAggregateMultiSourceDataNote: String
+
+  willImplementNewDataExchangeMethods: YesNoType
+  newDataExchangeMethodsDescription: String
+  newDataExchangeMethodsNote: String

```

Comment:
There is a field missing, there is a please describe field for willImplementNewDataExchangeMethods, and there is another for `Please describe any additional data exchange considerations.`

https://www.figma.com/design/qXhO12xSSkDF4LaToGKJoV/MINT-Blue-Sky-Ideas?node-id=2567-14638&node-type=frame&t=LW5WWJE9wgY2GRBR-0

Reply:
The corresponding fields are captured in the sections highlighted in this image
![image](https://github.com/user-attachments/assets/5afc90b9-153a-4c31-81d5-cbaa027938f8)

Reply:
![image](https://github.com/user-attachments/assets/7715443b-6b18-4b75-aaba-5d3a91ba9082)

Yeah -- looks like this should all be captured on this page as far as I can see @StevenWadeOddball -- is there something else from this page of the Figma that is missing, or do these fields cover it?

Reply:
Forgot to mark `isDataExchangeApproachComplete` in there too for that final checkbox, but that's there!

Reply:
Resolving this as it appears to be a false positive, we can re-open the issue if we find that there is something missing

Reply:
Thanks @OddTomBrooks and @ClayBenson94 , my apologies I missed the notification for this, but that Is my mistake. Thanks for clarifying!